src/calendar-heatmap.component.js

```diff
@@ -1223,12 +1241,7 @@ export class CalendarHeatmap extends Component {
       .attr('height', () => {
         return Math.min(dayScale.bandwidth(), this.settings.max_block_height);
       })
-      .attr('fill', (d) => {
-        let color = scaleLinear()
-          .range(['#ffffff', this.props.color])
-          .domain([-0.15 * max_value, max_value]);
```

Comment:
lower bound here is `-.015 * xyz` but you are using `min_value`

Reply:
In my understanding why the creator of this package did so because he wasn't calculating the minimum value there, I think that's why he chosen it. Since we can calculate minimum value, So i used it @maneetgoyal sir.

Reply:
- create a new issue in our repo, revert the change
- create a new issue in the original repo, ask the creator

Reply:
@maneetgoyal sir we haven't merged the code yet related to this change, no new issue is needed in `our repository` sir for reverting the code.

Reply:
>no new issue is needed

it is needed to use `min` in the future (if the creator say its okay)